docs/output-kafka.asciidoc

```diff
@@ -328,9 +340,9 @@ Kafka down, etc).
 A value less than zero is a configuration error.
 
 Starting with version 10.5.0, this plugin will only retry exceptions that are a subclass of
-https://kafka.apache.org/25/javadoc/org/apache/kafka/common/errors/RetriableException.html[RetriableException]
+https://kafka.apache.org/{kafka_client_doc}/javadoc/org/apache/kafka/common/errors/RetriableException.html[RetriableException]
 and
-https://kafka.apache.org/25/javadoc/org/apache/kafka/common/errors/InterruptException.html[InterruptException].
+https://kafka.apache.org/{kafka_client_doc}/javadoc/org/apache/kafka/common/errors/InterruptException.html[InterruptException].
```

Comment:
This link using `/25/` is from a recent PR.  Seems like we should standardize it, too. WDYT?

Reply:
:+1: most definitely - for correctness these should point to `/24` till we upgrade the client